lib/cocoapods/installer/installation_options.rb

```diff
@@ -109,6 +109,7 @@ def hash
       option :warn_for_multiple_pod_sources, true
       option :share_schemes_for_development_pods, false
       option :disable_input_output_paths, false
+      option :preserve_file_structure, false
```

Comment:
could you rename this to something that makes it clear it's meant for pod sources? maybe `preserve_pod_file_structure`